test/test_decomp.py

```diff
@@ -431,6 +431,11 @@ def test_unsupported(t):
     return any(test_unsupported(x) for x in itertools.chain(flat_args, flat_kwargs))
 
 
+core_not_decomposed = {
```

Comment:
No you shouldn't need to skip any tests. Just don't register the aliases to maximum as decompositions.

Reply:
Or are the tests still failing now?

Reply:
Yes, if I don't register the decompositions, the tests will still fail. But it makes sense since no decomposition occurs. I think the only way to prevent it is to skip the tests.

Reply:
My understanding is that since even though `max.other` doesn't have a decomposition registered, it will still be tested since other variants of `max` have decompositions. If `max.other` is tested, then `check_decomposed()` will fail since `maximum` gets called instead.

Reply:
What error do you get?

Reply:
```
AssertionError: False is not true : aten.max was not decomposed, saw calls for: aten.maximum.default. If your op is  CompositeImplicitAutograd you should skip this test by updating CROSS_REF_EXCLUDE_SET.

To execute this test, run the following from the base repo dir:
     python test/test_decomp.py -k test_quick_max_binary_cpu_uint8
```

Reply:
Hrm okay, I was expecting it to say `aten.max.other` but if it isn't checking at the overload level then I guess we do need to skip.